app/pathogen/arbovirus/dashboard/(map)/MapArbovirusFilter.tsx

```diff
@@ -1,18 +1,17 @@
-import { Card, CardContent, CardFooter, CardHeader } from "@/components/ui/card";
-import { ArboActionType, ArboContext } from "@/contexts/arbo-context/arbo-context";
+import { ArboActionType, ArboContextType } from "@/contexts/arbo-context/arbo-context";
 import { Checkbox } from "@/components/ui/checkbox";
-import { useContext } from "react";
 import { pathogenColorsTailwind } from "./ArbovirusMap";
 import { Button } from "@/components/ui/button";
+import SectionHeader from "@/components/customs/SectionHeader";
 
 interface MapArbovirusFilterProps {
-  records: unknown[];
   className?: string;
+  state: ArboContextType;
+  data: any;
 }
```

Comment:
This component now takes in just the exact same data sources as the other filters: solves the sync problem WITHOUT me having to do any really annoying stuff.  Always good.